README.md

```diff
@@ -105,41 +105,22 @@
 
 ### Build Tools for Windows
 
-1) Compiler (Choose one)
+1) Compiler for IA32/X64 (Choose one)
 
-    a) [Visual Studio 2022](https://visualstudio.microsoft.com/vs/older-downloads/)
+    a) [Visual Studio 2022](https://visualstudio.microsoft.com/vs/older-downloads/), [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/), [Visual Studio 2015](https://visualstudio.microsoft.com/vs/older-downloads/)
 
-    b) [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/)
-
-    c) [Visual Studio 2015](https://visualstudio.microsoft.com/vs/older-downloads/)
-
-    d) [LLVM](https://llvm.org/) (LLVM13)
+    b) [LLVM](https://llvm.org/) (LLVM13)
     - Install [LLVM-13.0.0-win64.exe](https://github.com/llvm/llvm-project/releases/tag/llvmorg-13.0.0). Please change LLVM install path to C:/LLVM, and add LLVM in PATH environment.

```

Comment:
Probably a separate issue, but since Ubuntu has moved to LLVM 14, Windows should as well.

Reply:
Let's keep it separate. It is not related to readme content move.